pyuvdata/uvdata/uvh5.py

```diff
@@ -474,30 +472,66 @@ def _read_header(
                 header["eq_coeffs_convention"][()]
             ).decode("utf8")
 
-        # We've added a new keyword that did not exist before, so check to see if it
-        # is in the header, and if not, mark the data set as being "regular" (i.e.,
-        # not a flexible spectral window setup).
+        # We've added a few new keywords that did not exist before, so check to see if
+        # it is in the header, and if not, mark the data set as being "regular" (e.g.,
```

Comment:
it is in -> any of them are in

Reply:
Fixed in 2e3a831bac5e94bbf80d5ab2db82c9eeb882b1a4.